src/puppetlabs/stockpile/queue.clj

```diff
@@ -265,7 +279,9 @@
   subtracting the 20 (UTF-8 encoded Basic Latin block) bytes reserved
   for internal use, there may be up to 235 bytes available for the
   metadata.  Of course how many Unicode characters that will allow
-  depends on their size when converted to UTF-8."
+  depends on their size when converted to UTF-8.  On error, may throw
+  the ::path-cleanup-failure-after-error exception documented in
+  durably-establish."
```

Comment:
Good point; I've included the docs in the create and store docstrings and elaborated on how we think callers might want to handle invocations.